cves/CVE-2002-1593.yml

```diff
@@ -47,13 +55,16 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: 618f31fcd3ab951204eeb98661c37ed1e765e316
-     note:
+  - commit: 618f31fcd3ab951204eeb98661c37ed1e765e316
+    note: There was a null pointer check added to stop this vulnerability.
 vccs:
   - commit: 8b00d096a6218d883a4ab69e546bb89f3f8c6290
-    note:
-  - commit:
-    note:
+    note: |
+      This is when the vulnerability was introduced there was no check
+      if the vsn_hooks pointer was null. It was assumed that get_option value
+      associated with the vsn_hooks existed and if vsn_hooks was null it would
+      error out, but that is not the case if this vulnerability exists.
```

Comment:
This might sound better if everything after the 'but' was made into a new sentence.